src/main/java/uk/gov/justice/digital/zone/Zone.java

```diff
@@ -53,4 +54,9 @@ protected void appendToDeltaLakeTable(Dataset<Row> dataFrame, String tablePath)
         logger.info("Append completed successfully");
     }
 
+    protected Dataset<Row> createEmptyDataFrame(Dataset<Row> dataFrame) {
```

Comment:
Since this is specific to the structured zone it could stay there for now.

Reply:
no Im using this in CuratedZone too